ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/events/PostCommitValidation.scala

```diff
@@ -113,8 +114,13 @@ private[appendonlydao] object PostCommitValidation {
       def foldInformees[T](tx: CommittedTransaction, init: T)(
           f: (T, Party) => T
       ): T =
-        tx.fold(init) { case (accum, (_, node)) =>
-          node.informeesOfNode.foldLeft(accum)(f)
+        tx.fold(init) {
+          case (accum, (_, node: LFNode.GenActionNode[_, _])) =>
+            node.informeesOfNode.foldLeft(accum)(f)
+          case (accum, (_, _: LFNode.NodeRollback[_])) =>
+            // TODO https://github.com/digital-asset/daml/issues/8020
```

Comment:
Same comment above, there are no informees so this is correct afaict.

Reply:
Same answer than above.

Reply:
This is the counterpoint to implicit party allocation that validates all parties have been allocated. Parties in rolled back nodes also need to be taken into account.